rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,134 @@
+# RFC: Communicating new package structure for `@fluentui/*`
+
+**Note that this RFC focuses solely on the package structure from a consumer perspective, focused primarily on v7/8 and converged packages. It does not deal with folder structure or northstar packages.**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+Currently, we have a lot of different component packages, and it's not clear to consumers what the purpose of each one is or whether it's production-ready.
+
+There are a few aspects to the proposed solution:
+
+- Add badges (or similar) to package readmes to clarify GA/preview/dev status
+- "Converged" component packages only contain convergence work
+  - Move old (v8) component implementations from `react-{checkbox,link,slider,tabs,toggle}` back to `react-internal`
+  - Move prototype "converged" components to the package roots and get rid of `/next` folders
```

Comment:
**q:** 
regarding `prototype "converged" components`
- Which components/packages belong to this group? 
- These components will be "redone" through the new convergence process specs/open ui/implementation workflow or they are good to go?

Reply:
It's referring to the packages listed in the previous bullet point: `react-{checkbox,link,slider,tabs,toggle}`. The "Background" and "Related code moves" sections go into this more (unfortunately the background table is a bit hard to read in the markdown text view).

Basically, these packages currently export the old/v8 component implementation by default. We've decided that was not the right approach and will be moving the old implementations back to `@fluentui/react` shortly. The reason I called the newer implementations in those packages "prototype converged" is because they're all still works in progress and need to be updated to use whatever final patterns we decide on.

Reply:
Agreed, flatten /next into the package root. While we're iterating hard on convergence let's not save previous iterations. We will always have the alpha/beta package releases that may have happened along the way.